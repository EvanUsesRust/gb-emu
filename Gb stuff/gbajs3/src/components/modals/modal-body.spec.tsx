import { render, screen } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import { ModalBody } from './modal-body.tsx';

describe('<ModalBody />', () => {
  it('renders children', () => {
    render(<ModalBody>Test Footer Content</ModalBody>);

    expect(screen.getByText('Test Footer Content')).toBeInTheDocument();
  });

  it('container matches snapshot', () => {
    render(<ModalBody>Test Footer Content</ModalBody>);

    expect(screen.getByText('Test Footer Content')).toMatchSnapshot();
  });
});
