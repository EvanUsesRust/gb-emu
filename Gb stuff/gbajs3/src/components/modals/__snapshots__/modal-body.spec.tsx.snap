// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<ModalBody /> > container matches snapshot 1`] = `
.c0 {
  padding: 1rem;
  overflow-y: auto;
}

<div
  class="c0"
>
  Test Footer Content
</div>
`;
